.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v2-lint-deps
+            - v3-lint-deps
```

Comment:
What about adding `{{ checksum "tox.ini" }}` instead?
Makes sense in general, too.